kit/score/score.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.33.0
+// 	protoc-gen-go v1.32.0
```

Comment:
Perhaps you should upgrade your `protoc-gen-go` plugin to avoid going backwards in version numbers.